src/code/audio_heap.c

```diff
@@ -10,32 +10,40 @@ void AudioHeap_DiscardSampleCaches(void);
 void AudioHeap_DiscardSampleBank(s32 sampleBankId);
 void AudioHeap_DiscardSampleBanks(void);
 
-f32 func_800DDE20(f32 arg0) {
-    return 256.0f * gAudioContext.audioBufferParameters.unkUpdatesPerFrameScaled / arg0;
+/**
+ * Effectively scales updatesPerFrameInv by the reciprocal of scaleInv
+ * updatesPerFrameInvScaled is just updatesPerFrameInv scaled down by a factor of 256.0f
+ * i.e. (256.0f * gAudioContext.audioBufferParameters.updatesPerFrameInvScaled) is just updatesPerFrameInvScaled.
```

Comment:
I don't understand this comment. Did you mean `updatesPerFrameInv` at the end there?

Also I suggest using ``` quotes around variable names in there to make them stand out a bit more, and no need for the `gAudioContext` access imo

Reply:
Yeah, `updatesPerFrameInv` is what it's suppose to be, that's just a mistake. I'll remove the `gAudioContext` and add the quotes